src/features/commands/buttons/removeAll.js

```diff
@@ -0,0 +1,38 @@
+const {
+    Client
+} = require('discord.js'),
+    db = require('../../../db'),
+    perspective = require('../../../api/perspective');
```

Comment:
Unnecessary dependencies
```suggestion
const {
    Client
} = require('discord.js'),
    db = require('../../../db');
```